### PR TITLE
client requested additional changes

### PR DESCRIPTION
**Pull request by ksaini75**
Client requested some additional changes to the service, about section, footer and contact page.

### CODE REVIEW

File: src/components/AboutPage/AboutPage.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ const AboutPage = () => {
               </div>
               <div>
                 <p className="about-text text-center mb-5" >
-                  We are glad to offer you a wide range of residential electrical services - from changing fixtures, upgrading panels, garden lighting to outdoor outlets and much more. 
+                  We are glad to offer you a wide range of residential and commercial electrical services - from changing fixtures, upgrading panels, garden lighting to outdoor outlets and much more. 
                   <br/>
```
Comment (ksaini75) [inline src/components/AboutPage/AboutPage.js:121]: Client requested to add "commercial after residential"

File: src/components/ContactPage/ContactPage.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ const ContactPage = () => {
             }, function (error) {
                 swal({
                     title: `Message not sent!`,
-                    text: `Please email me: KCelectrialcontractorllc@gmail.com`,
+                    text: `Please email me: kcelectricalcontractorllc@gmail.com`,
```
Comment (ksaini75) [inline src/components/ContactPage/ContactPage.js:77]: Updated the correct email address

File: src/components/ContactPage/ContactPage.js [hunk 3]
```diff
@@ -108,12 +108,12 @@ const ContactPage = () => {
                 <Row className="mt-5 mb-5">
                     <Col md={5} className="text-center">
                         <FaPhoneAlt className="contact-icons" />
-                        <h5 className="contact-info mt-3"> (347)-997-0315</h5>
+                        <h5 className="contact-info mt-3"><a href="tel:3479970315"> (347)-997-0315</a></h5>
```
Comment (ksaini75) [inline src/components/ContactPage/ContactPage.js:111]: Client requested there should be a clickable link for user to call the number and email as well for the contact page

File: src/components/FooterPage/Footer.js [hunk 4]
```diff
@@ -10,16 +10,21 @@ class Footer extends Component {
...
                             <a className="insta-icon" href="https://www.instagram.com/kcelectricalcontractorllc/?igshid=13e62ll0r2ahi" target="_blank" rel="noopener noreferrer">
                                 <FaInstagram />
                             </a>
                         </p>
                     </Col>
+                    <Col md={4} className="mr-auto">
+                        <p>
+                        We also accept Zelle, Venmo and most major credit cards.
+                        </p>
+                    </Col>
```
Comment (ksaini75) [inline src/components/FooterPage/Footer.js:24]: Client requested to add the company accepts Zelle

File: src/components/ServicePage/ServicePage.js [hunk 5]
```diff
@@ -44,6 +44,7 @@ const ServicePage = () => {
                             <p> <GiLightBulb /> Renovations or additions Indoor or outdoor lighting</p>
                             <p><GiLightBulb /> Inspections</p>
                             <p><GiLightBulb /> Wiring installation/Wiring Repairs</p>
+                            <p><GiLightBulb/> Tesla Charger Installation</p>
                         </Col>
```
Comment (ksaini75) [inline src/components/ServicePage/ServicePage.js:48]: added an additional service

**Review by MrGDS2 — APPROVED** [review]
These changes look good. Thank you!

**Pull request merged**